entries/n/ngpvan.com.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Site Name": {
+    "domain": "ngpvan.com",
+    "img": "ngpvan.com.png",
```

Comment:
```suggestion
```

You can use this SVG instead.
![NGP VAN](https://github.com/user-attachments/assets/75551e2f-c197-4de8-a10d-6039ef328e5b)